pkg/util/log/friendly.go

```diff
@@ -0,0 +1,14 @@
+package log
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"reflect"
+	"runtime"
+)
+
+// FriendlyName returns a "friendly" stringified name of the given func.
+func FriendlyName(f interface{}) string {
```

Comment:
ick - this is a bit like https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common .
If we want to promote reuse of this function across other packages, it should be named better.  In fact, I think this should be an unexported function in util/steps until we know we need it elsewhere.